src/main/java/net/syscon/elite/repository/jpa/model/LivingUnitProfile.java

```diff
@@ -0,0 +1,74 @@
+package net.syscon.elite.repository.jpa.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JoinColumnOrFormula;
+import org.hibernate.annotations.JoinColumnsOrFormulas;
+import org.hibernate.annotations.JoinFormula;
+import org.hibernate.annotations.NotFound;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import java.io.Serializable;
+
+import static org.hibernate.annotations.NotFoundAction.IGNORE;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@IdClass(LivingUnitProfile.PK.class)
+@Table(name = "LIVING_UNIT_PROFILES")
+public class LivingUnitProfile {
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class PK implements Serializable {
+
+        @Column(name = "LIVING_UNIT_ID")
+        private Long livingUnitId;
+
+        @Column(name = "DESCRIPTION")
+        private String description;
+
+        @Column(name = "AGY_LOC_ID")
+        private String agencyLocationId;
+
+        @Column(name = "PROFILE_ID")
+        private Long profileId;
+
+    }
+
+    @Id
+    @Column(name = "LIVING_UNIT_ID", nullable = false)
+    private Long livingUnitId;
+
+    @Id
+    @Column(name = "AGY_LOC_ID", nullable = false)
+    private String agencyLocationId;
```

Comment:
See agency comment above.  We have an object to model this.

Reply:
Same as above.